### ISSUE TITLE
Project file is edited back to the standard template

### ISSUE THREAD

**Issue by rubendibattista** [tmuxinator/tmuxinator#378]
I'm trying to create my fist project and starting playing with tmuxinator:

`mux new foo`

I make my edits to the standard template then save `:w`.

`mux start foo` works correctly

I want to edit the project.

`mux open foo`

The project now opened by vim is the standard one, not the one I edited. Ok, I go back without saving `:q!`

`cat ~/.tmuxinator/foo.yml`

Prints the default template and not the one I edited. 

What am I doing wrong?

Environment
------

- OSX 10.11.3
- tmux 2.1 (NHDaly/tmux-scroll-copy-mode plugin)
- oh-my-zsh (w/ tmuxinator and tmux plugins)

```
which mux
/usr/local/bin

which tmux
tmux: aliased to _zsh_tmux_plugin_run
```

**Comment by rubendibattista** [comment]
I think the issue was related to https://github.com/robbyrussell/oh-my-zsh/issues/518.

Executing `exec zsh` should solve the problem.

**Comment by ethagnawl** [comment]
@rubendibattista Which version of Tmuxinator are you using? Your problem sounds like #354, which was fixed in version 0.7.1.

**Comment by rubendibattista** [comment]
True!

I also updated tmuxinator 
`gem update tmuxinator`

So it was the version.

Thanks @ethagnawl !
